### ISSUE TITLE
lr, wd in multi node training

### ISSUE THREAD

**Issue by poodarchu** [facebookresearch/detectron2#1322]
If you do not know the root cause of the problem / bug, and wish someone to help you, please
post according to this template:

## Instructions To Reproduce the Issue:

1. what changes you made (`git diff`) or what code you wrote

I use 4 machines to train regular retinanet using config:  https://github.com/facebookresearch/detectron2/blob/master/configs/COCO-Detection/retinanet_R_50_FPN_1x.yaml, and enlarge lr, batch size by 4 times, and shrink steps, iters and weight decay to its 1/4

2. what expected:
get the same 36.5 mAP as training using one machine.

3. what you observed (including __full logs__):
```
only get 36.1 after training with 4 machines.
```
4.  then I keep weight decay unchanged and only change lr, batchsize, steps and iters, the training will then soonly encounter Nan. 

Is there anything I configured wrongly?
